book_source/02_demos_tutorials_workflows/01_install_pecan.Rmd

```diff
@@ -136,6 +136,8 @@ This will not go into much detail about about how to use Docker -- for more deta
        ```{r, echo=FALSE, fig.align='center'}
        knitr::include_graphics(rep("figures/env-file.PNG"))
        ```
+       
+       Setting `PECAN_VERSION=develop` indicates that you want to run the bleeding-edge `develop` branch, meaning it may have bugs. To go ahead with the stable version you may set `PECAN_VERSION=latest` or `PECAN_VERSION=<release-number>` (For example `1.7.0`). [Here](https://github.com/pecanproject/pecan/releases) are the details about all the PEcAn releases.
```

Comment:
```suggestion
       Setting `PECAN_VERSION=develop` indicates that you want to run the bleeding-edge `develop` branch, meaning it may have bugs. To go ahead with the stable version you may set `PECAN_VERSION=latest` (master branch on github) or `PECAN_VERSION=<release-number>` (For example `1.7.0`).  You can look at the list of all  [releases](https://github.com/pecanproject/pecan/releases) of PEcAn to see what options are availble.
```